packages/ts-morph/src/tests/projectTests.ts

```diff
@@ -43,7 +43,7 @@ describe(nameof(Project), () => {
             fileSystem.writeFileSync("/test/file.ts", "");
             fileSystem.writeFileSync("/test/test2/file2.ts", "");
             const project = new Project({ tsConfigFilePath: "tsconfig.json", compilerOptions: { rootDir: "/test/test2" }, fileSystem });
-            expect(project.getSourceFiles().map(s => s.getFilePath()).sort()).to.deep.equal(["/test/test2/file2.ts"].sort());
+            expect(project.getSourceFiles().map(s => s.getFilePath()).sort()).to.deep.equal(["/otherFile.ts", "/test/file.ts", "/test/test2/file2.ts"].sort());
```

Comment:
`include` is `["**/*"]` by default, so `/otherFile.ts` and `/test/file.ts` should be included too. And tsc should fail successfully.

> error TS6059: File '/otherFile.ts' is not under 'rootDir' '/test/test2'. 'rootDir' is expected to contain all source files.

Reply:
Learn something new every day 😄